utils/frame/remote-externalities/src/lib.rs

```diff
@@ -115,27 +115,28 @@ use sp_core::{
 use codec::{Encode, Decode};
 use jsonrpsee_http_client::{HttpClient, HttpConfig};
 
+use sp_runtime::traits::Block as BlockT;
+
+// TODO: Make KeyPair generic
```

Comment:
hmm? really? :D

Reply:
The original todo was "make these two generic." and I assumed it was meant for both the hash and the keypair. But I only made the hash generic, so I re-added the todo for the keypair, so it wont get lost.

Reply:
ahh okay, we can remove this. Storage key and pair are already Vec<u8>, generic enough.

Reply:
```suggestion
```